packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -102,6 +103,10 @@ function convertSummaryIntoContainerSnapshot(createNewSummary: ISummaryTree) {
         throw new Error("App and protocol summary required for create new path!!");
     }
     const documentAttributes = getDocAttributesFromProtocolSummary(protocolSummary);
+    // Currently for the scenarios we have we don't have ops in the detached container. So the
+    // sequence number would always be 0 here. However odsp requires to have at least 1 snapshot.
+    assert(documentAttributes.sequenceNumber === 0, "Sequence No for detached container snapshot should be 0");
```

Comment:
I think @jatgarg removed this assert in later iterations, so likely that's incorrect merge (or missing change?)

Reply:
Yes, I modified this asseret. Could you please change it to that?

Reply:
Looks like this actually would cover the assert issue as well oops